components/prism-typoscript.js

```diff
@@ -0,0 +1,31 @@
+    }, {
+        pattern: /pagets__.*(?:\s|=|.)/g
+    }],
+    'function': [{
+        pattern: /@import/g,
+        greed: true,
+    }, {
+        pattern: /{[$#].*}|{(?:register|field|cObj):.*}|(?:TSFE|file):.*\n?\s?/g
```

Comment:
1. The `.*` in `{[$#].*}` and `{(?:register|field|cObj):.*}` is greedy. Does that mean that it is allowed to match `{` and `}`?
2. Why the `\n?\s?`?

Reply:
I've reworked this too, will add it asap.